### ISSUE TITLE
500 Internal Server Error

### ISSUE THREAD

**Issue by kevwaddell** [wp-sync-db/wp-sync-db#53]
Hi
I tried to connect to my live server to pull down a current copy to my local machine using the API key request and I got the following error.
---------
Unable to connect to the remote server, please check the connection details - 500 Internal Server Error (#129 - scope: ajax_verify_connection_to_remote_site)

When logged in to my live site I tried to reset the API key but when I do I get an alert message.
---------
An error occurred when trying to generate the API key. 
Please see the Help tab for details on how to request support. (#105)

Also when I go to the help tab on the live site, I get the following error.
--------
An error occurred when trying to update the debug log. Please contact support. (#133)

**Comment by jondueck** [comment]
I've also been having this 500 Error #129. I was able to reset the API key and reconnect, but it did not fix the issue.

**Comment by lesteak** [comment]
Have spent some time also trying to fix this and found that removing the php_value configuration from .htaccess resolved the issue:
php_value upload_max_filesize
php_value post_max_size
php_value memory_limit

**Comment by trialotto** [comment]
Guys,

This is not really a bug. 

In essence, most of the cases, that result in this #129 error, are related to 

- memory shortage: a lack of free memory (just check it with free -hm command)
- mysql issues: cpu or memory overusage (or even slow queries, as is common with WP)

and the most easy ways to resolve the #129 error are

a) have some patience, in order to allow MySQL to return to a normal state, 

Note: normally, one would optimize MySQL resource usage, but if you have the error, you are in fact too late and/or the MySQL tuning was not sufficient

b) close some WordPress panels,if you are logged into many, 

c) just restart mysql daemon (this is the "last resort"). 

In short, do not be worried if you get this #129 error, it can be easily solved by extra memory. 

Hope this helps a bit. 

Regards....

**Comment by jan1980** [comment]
Which are some tips so we can increase memory? Should we write something on .htaccess or wp-config.php or php.ini? Thanks!
